distribution/sdk/artemis-sandbox/src/main/java/sdk.example.route/AmqpProducerConsumerExample.java

```diff
@@ -57,7 +57,6 @@ public void configure() throws Exception {
     onException(Throwable.class).logStackTrace(true);
 
     createCamelContext();
-    CamelContext camelContext = getContext();
```

Comment:
❗️ I don't think you simply want to remove that line. This is an example and although the example is not using it, The idea is that real code would. We need to suppress here I think.